### PR TITLE
#170947540 Coverage Integration

### PR DESCRIPTION
**Pull request by Baraka-Mugisha**
#### What does this PR do?
- Integrate Coveralls

#### Description of Task to be completed?
- install coveralls
- add a coveralls badge on ReadMe
- add a .coveralls.yml

#### How should this be manually tested?
- go to ReadMe and view the badge

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#170947540](https://www.pivotaltracker.com/story/show/170947540)

#### Screenshots (if appropriate)
Screen Shot 2020-02-07 at 15 37 33

#### Questions:
- N/A

### CODE REVIEW

**Review by StevenDerrick — DISMISSED** [review]

**Review by Ntare22 — DISMISSED** [review]

**Comment by Mnickii** [comment]
reabase

**Review by StevenDerrick — DISMISSED** [review]

**Review by bbaime98 — DISMISSED** [review]

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: McVFxLuAXHsE4OgdFvXxybGGDwc4Y1Lrc
```
Comment (Mnickii) [inline .coveralls.yml]: Let's not commit any tokens

File: package.json [hunk 2]
```diff
@@ -24,11 +24,12 @@
...
     "chai-http": "^4.3.0",
-    "mocha": "^7.0.1",
-    "nyc": "^15.0.0",
+    "coveralls": "^3.0.9",
     "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.20.1",
-    "eslint-config-airbnb-base": "^14.0.0"
+    "mocha": "^7.0.1",
+    "nyc": "^15.0.0"
```
Comment (Mnickii) [inline package.json:46]: where have we used this?

**Review by Mnickii — CHANGES_REQUESTED** [review]
Include a screenshot of the coverage on coveralls.io. We need to see a collected coverage after testing the application. The task is more than just including the badge

File: package.json (hunk 2 above)
Comment (Baraka-Mugisha) [reply package.json:46]: Mnickii, I pushed a script for running coverage with a mocha reporter recently.

**Review by Mnickii — APPROVED** [review]

File: .coveralls.yml (hunk 1 above)
Comment (Baraka-Mugisha) [reply .coveralls.yml]: Yeah, thanks. It is ignored now.

**Pull request merged**